src/app/assessment/assessment.component.ts

```diff
@@ -787,4 +763,37 @@ export class AssessmentComponent extends RouterEnter {
     }
   }
 
+  doAssessmentDoReview() {
```

Comment:
let's rename it to `doAssessmentDoReviewStatus` so that we knows that it returns the status